satellite/rewards/rewards.go

```diff
@@ -69,12 +69,13 @@ const (
 type OfferStatus int
 
 const (
-	// Done is a default offer status when an offer is not being used currently
-	Done = OfferStatus(iota)
+
+	// Active is a offer status when an offer is currently being used
+	Active = OfferStatus(iota)
```

Comment:
Doesn't this require a database migration?

Reply:
I changed "done" back to the first one